src/components/LicenseFilters/LicenseFilters.js

```diff
@@ -71,11 +72,67 @@ export default class LicenseFilters extends React.Component {
     );
   }
 
+  renderOrganizationFilter = () => {
+    const activeFilters = this.props.activeFilters.orgs || [];
+
+    return (
+      <Accordion
+        closedByDefault
+        displayClearButton={activeFilters.length > 0}
+        header={FilterAccordionHeader}
+        label={<FormattedMessage id="ui-licenses.agreements.organizations" />}
+        onClearFilter={() => {
+          this.props.onChange({ name: 'orgs', values: [] });
+          this.props.onChange({ name: 'role', values: [] });
+        }}
+      >
+        <OrganizationSelection
+          path="licenses/org"
+          input={{
+            name: 'license-orgs-filter',
+            onChange: value => this.props.onChange({ name: 'orgs', values: [value] }),
+            value: activeFilters[0] || '',
+          }}
+        />
+      </Accordion>
+    );
+  }
+
+  renderRoleLabel = () => {
+    const roles = get(this.props.resources.orgRoleValues, ['records'], []);
+    const dataOptions = roles.map(role => ({
+      value: role.id,
+      label: role.label,
+    }));
+
+    const orgFilters = this.props.activeFilters.orgs || [];
+    const activeFilters = this.props.activeFilters.role || [];
+
+    return (
+      <Accordion
+        closedByDefault
+        displayClearButton={activeFilters.length > 0}
+        header={FilterAccordionHeader}
+        label={<FormattedMessage id="ui-agreements.settings.orgRoles.orgRole" />}
```

Comment:
In general, don't rely on another module's translations keys like you do here relying on ui-agreements. 

(The exception could be using stripes-core's or stripes-component's generic strings.)